internal/config/config.go

```diff
@@ -271,6 +272,24 @@ func (c *Config) Dump(msg string) {
 	}
 }
 
+// YamlExtension tries to find the correct extension for a YAML file
+func YamlExtension(path string) string {
```

Comment:
@placintaalexandru Perhaps leverage filepath.Ext to check for file mime? 
Inverting the logic as more than likely ".yml" will be there?
Also we could just prepend once trim ie path + ".yaml" vs strings.Join that avoid allocations?

Reply:
hello @derailed 
`filepath.Ext` does not check any mime type, it looks at the extension of the file

Reply:
Hi @placintaalexandru - Right but I think its a better mechanic to pull out the extension.

Reply:
yes, I agree. I used that after I saw what it does